tests/js/arrows/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -686,8 +686,9 @@ jest.mock(
     }
 );
 
-fooooooooooooooooooooooooooooooooooooooooooooooooooo((action) => (next) =>
-  dispatch(action)
+fooooooooooooooooooooooooooooooooooooooooooooooooooo(
+  (action) =>
+  (next) => dispatch(action)
 );
```

Comment:
This seems not pretty.

Reply:
Maybe no break before first level parameters more reasonabe.

Reply:
what do you mean? Should be like below?:
```js
fooooooooooooooooooooooooooooooooooooooooooooooooooo((action) =>
  (next) => dispatch(action)
);
```

Reply:
Yes, I mean this, but I'm not sure. Better idea?

Reply:
I'd prefer not breaking the chain if it fits on one line:
```js
fooooooooooooooooooooooooooooooooooooooooooooooooooo(
  (action) => (next) => dispatch(action)
);
```
And if the chain is long enough, then:
```js
fooooooooooooooooooooooooooooooooooooooooooooooooooo(
  (action) =>
    (next) =>
    (next) =>
    (next) =>
    (next) =>
    (next) =>
    (next) => 
    dispatch(action)
);
```
This is similar to how Prettier formats binary expressions in call arguments.

Reply:
In that case, assign print like this?

```js
const foo = (action) =>
  (next) =>
  (next) =>
  (next) =>
  (next) =>
  (next) =>
  (next) => 
  dispatch(action)
```

Reply:
Assignments are complicated. To make the chain look uniform and to avoid dealing with edge cases (#2482), it's better to break after `=`.

```js
const foo =
  (action) =>
  (next) =>
  (next) =>
  (next) =>
  (next) =>
  (next) =>
  (next) => 
  dispatch(action)
```

Reply:
Thank you. I agree with thorn0's suggestions.